app/js/TabWrangleOption.js

```diff
@@ -0,0 +1,71 @@
+/* @flow */
+
+import React from 'react';
+
+type Props = {
+  selectedOption?: string,
+  onChange?: () => void
+};
+
+const OPTIONS = [
+  { name: 'withDupes', text: 'With Duplicates' },
+  { name: 'exactURLMatch', text: 'Exact URL match' },
+  { name: 'hostnameAndTitleMatch', text: 'Hostname and Title match' },
+];
+export default class TabWrangleOption extends React.Component {
+  constructor(props: Props) {
+    super(props);
+
+    this.state = { selectedOption: this.props.selectedOption };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(e) {
+    this.setState({
+      selectedOption: e.target.value,
+    });
+
+    this.props.onChange(e);
```

Comment:
This is passing the new value up the hierarchy, and it's being passed in via props. Why have `state` in this component at all?

Reply:
I keep the state to reflect which radio button is pressed. If there is a better way to achieve the same, I'm happy to learn about it.